backend/app/views/spree/admin/store_credits/_update.js.erb

```diff
@@ -0,0 +1,5 @@
+<% if success = flash.discard(:success) %>
```

Comment:
Like this?

```
var flash = { success: <%= flash.discard(:success).to_json %>, error: <%= flash.discard(:error).to_json %> };
if(flash['success']) { show_flash('succcess', flash['success']); }
if(flash['error']) { show_flash('error', flash['error']); }
```

Chatting with Richard on this, want to make sure we understand your question correctly.